core/src/main/java/nl/nn/adapterframework/core/Adapter.java

```diff
@@ -603,8 +604,9 @@ public PipeLineResult processMessage(String messageId, Message message, PipeLine
 					objectInError = this;
 				}
 			}
-			result.setResult(formatErrorMessage(msg, t, message, messageId, objectInError, startTime));
-			messageLogResult(messageId, result, null);
+			try (final CloseableThreadContext.Instance ctc = LogUtil.getThreadContext(this, messageId, pipeLineSession)) {
+				result.setResult(formatErrorMessage(msg, t, message, messageId, objectInError, startTime));
+			}
```

Comment:
Is dit nu niet dubbel? formatErrorMessage draait al in een `CloseableThreadContext`.

Reply:
De CloseableThreadContext inf formatErrorMessage is niet compleet, want die heeft geen PipeLineSession. Het is dus daar meer een soort last resort, om toch iets te hebben als het buiten niet goed gezet is.